pr-checks/checks/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Check } from '../Check'
+import { MilestoneCheck, MilestoneCheckConfig } from './MilestoneCheck'
+
+export type CheckConfig = {
```

Comment:
You can do:

```
export type CheckConfig = MilestoneCheckConfig & {
	type: string
}
```

and then 

`checks.push(new MilestoneCheck(checkConfig))`

Reply:
Fixed in a similar way. Going to start working on a new check in a separate PR so will need to update this then.